.github/workflows/pr-ci.yml

```diff
@@ -1,19 +1,35 @@
-name: Plugins Platform PR - CI
+name: Plugins - CI / CD
 
 on:
   push:
     branches:
       - main
   pull_request:
-    branches:
-      - main
+
+permissions: {}
 
 jobs:
-  ci:
-    name: CI
-    uses: grafana/plugin-ci-workflows/.github/workflows/ci.yml@main
+  cd:
+    name: CI / CD
+    uses: grafana/plugin-ci-workflows/.github/workflows/cd.yml@main # zizmor: ignore[unpinned-uses]
+    permissions:
+      contents: write
+      id-token: write
+      attestations: write
     with:
-      plugin-version-suffix: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.sha || '' }}
+      # Checkout/build PR or main branch, depending on event
+      branch: ${{ github.event_name == 'push' && github.ref_name || github.ref }}
+
+      # When pushing to "main", publish and deploy to "dev" (CD). For PRs, skip publishing and deploying (run CI only)
+      environment: ${{ (github.event_name == 'push' && github.ref_name == 'main') && 'dev' || 'none' }}
+
+      # Deploy provisioned plugin to Grafana Cloud
+      grafana-cloud-deployment-type: provisioned
+      argo-workflow-slack-channel: '#grafana-plugins-platform-ci'
```

Comment:
Should we use our own channel or this is a catch-all channel?

Reply:
This should be changed to your own slack channel. If you don't have one, I think it's fine to have this set to the default `'#grafana-plugins-platform-ci'` for now (dev/ops), but should be changed to something different before going to prod